android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -52,7 +52,7 @@
 		"name": "Library",
 		"hurryCostModifier": 25,
 		"maintenance": 1,
-		"uniques": ["[+1 Science] Per [2] Population in this city"],
+		"uniques": ["[+1 Science] Per [2] Population [in this city]"],
```

Comment:
Did you check this works? 'in this city' uniques need to be checked

Reply:
For all of the branch PR's, I checked every single policy unique again, just to be sure they all worked.
This one worked in the capital after building a library there, so I assumed it worked in all cities with a library